### PR TITLE
Fix small typos regarding the ssh address of the raspberry

### PR DESCRIPTION
**Pull request by Alevale**

### CODE REVIEW

**Comment by Alevale** [comment]
@marcone While trying to ssh into the raspberry I discovered that the docs were wrong in some places, and that the actual "namespace" was `pi@teslausb.local` and not the other specified.
If this is incorrect let me know.

File: doc/GetShellWithoutMonitorOnLinux.md [hunk 1]
```diff
@@ -31,6 +31,6 @@ These instructions will configure a Raspberry Pi so that you can proceed with th
     ```
     -- Files updated and ready for Wifi and SSH over USB --
     ```
-1. Eject the SD card safely, insert into your Pi, and reboot. If the Pi is connected over USB to your host, and/or if the Wifi setup went correctly, you should be able to `ssh pi@raspberrypi.local`. The default password is `raspberry`.
```
Comment (marcone) [inline doc/GetShellWithoutMonitorOnLinux.md:34]: This document is about doing a manual install of regular Raspbian (none of these steps are needed when using the prebuilt image), in which case the hostname is "raspberrypi" as shown here.

File: doc/GetShellWithoutMonitorOnWindows.md [hunk 2]
```diff
@@ -35,7 +35,7 @@
 1. Wait for something labeled "USB Ethernet/RNDIS Gadget" to appear.
 1. Enter this command in your PowerShell window:
     ```
-    ssh pi@raspberrypi.local
```
Comment (marcone) [inline doc/GetShellWithoutMonitorOnWindows.md:38]: ditto

**Comment by marcone** [comment]
No response. Closing.

**Pull request closed without merging**